src/InputField.js

```diff
@@ -127,9 +127,9 @@ class InputField extends React.Component {
           {React.cloneElement(InputChild, {
             'aria-label':
               !showLabel && inputPlaceholder ? inputPlaceholder : null,
-            mt: showLabel && -20,
+            mt: showLabel ? -20 : 0,
```

Comment:
Just noticed that the Input component itself doesn't reset its margin to `0` – is it worth making an update to that component instead?

Reply:
oh yeah that might be a better solution. I'll post an update tomorrow morning